### PR TITLE
[Windows CI] copy onnxruntime.dll to c++ test folder in windows

### PR DESCRIPTION
**Pull request by betterpig**
### PR types

Others
### PR changes

Others
### Describe

copy onnxruntime.dll to c++ test folder in windows. These tests need onnx to execute, but onnxruntime.dll can't be find even set add its path to ENV .

### CODE REVIEW

File: cmake/external/onnxruntime.cmake [hunk 1]
```diff
@@ -134,3 +134,18 @@ endif()
 add_library(onnxruntime STATIC IMPORTED GLOBAL)
 set_property(TARGET onnxruntime PROPERTY IMPORTED_LOCATION ${ONNXRUNTIME_LIB})
 add_dependencies(onnxruntime ${ONNXRUNTIME_PROJECT})
+
+function(copy_onnx TARGET_NAME)
+  # If error of Exitcode0xc000007b happened when a .exe running, copy onnxruntime.dll
+  # to the .exe folder.
+  if(WITH_TESTING AND NOT "$ENV{CI_SKIP_CPP_TEST}" STREQUAL "ON")
```
Comment (zhwesky2010) [inline cmake/external/onnxruntime.cmake]: 这里能否直接判断是否有这个 TARGET，有的话给这个TARGET加个POST_BUILD和DEPENDS

Comment (betterpig) [reply cmake/external/onnxruntime.cmake]: done

File: cmake/external/onnxruntime.cmake [hunk 2]
```diff
@@ -134,3 +134,15 @@ endif()
...
+
+function(copy_onnx TARGET_NAME)
+  # If error of Exitcode0xc000007b happened when a .exe running, copy onnxruntime.dll
+  # to the .exe folder.
+  if(TARGET ${TARGET_NAME})
+    add_custom_command(
+      TARGET ${TARGET_NAME}
+      POST_BUILD
+      COMMAND ${CMAKE_COMMAND} -E copy ${ONNXRUNTIME_SHARED_LIB}
+              ${CMAKE_CURRENT_BINARY_DIR} DEPENDS onnxruntime)
```
Comment (zhwesky2010) [inline cmake/external/onnxruntime.cmake:146]: 这个DEPENDS没另起一行吗

**Review by zhwesky2010 — APPROVED** [review]

File: cmake/external/onnxruntime.cmake (hunk 2 above)
Comment (betterpig) [reply cmake/external/onnxruntime.cmake:146]: 下次PR改过来~

**Pull request merged**